BUILD

```diff
@@ -226,6 +227,7 @@ cc_library(
     "src/core/json/json_common.h",
     "src/core/json/json_reader.h",
     "src/core/json/json_writer.h",
+    "src/core/proto/load_balancer.pb.h",
```

Comment:
That's precisely what we have. The problem found while introducing the change you proposed is that now the nanopb outputs xxx.pb.h to .../grpc/lb/v0, the same place the C++ proto compiler will (by virtue of the "package grpc.lb.v0" line in the proto). The C++ .h will then wreak havoc in all the C files that include and expect nanopb's C pb.h.

We either stop relying on the C++ generated code for the .proto in the tests (bummer, will have to augment the C LB proto API just for testing) or we output the nanopb generated code to a non-package-consistent directory (as before).